distribution/src/bin/elasticsearch

```diff
@@ -52,6 +52,10 @@ if [ -z "$LIBFFI_TMPDIR" ]; then
   export LIBFFI_TMPDIR
 fi
 
+if [ "$ES_DISTRIBUTION_TYPE" = 'docker' ] && [ -d '/usr/local/cloudflare-zlib/lib' ]; then
+  export LD_LIBRARY_PATH="/usr/local/cloudflare-zlib/lib:$LD_LIBRARY_PATH"
```

Comment:
Can't we set this in the docker image itself, or the docker entrypoint script?

Reply:
Good idea, I've changed this so that the `Dockerfile` sets an env var, and the `elasticsearch` script picks that up instead.